integration_tests/configs/nft_transfer.jsonnet

```diff
@@ -0,0 +1,27 @@
+local default = {
```

Comment:
```suggestion
local ibc = import 'ibc.jsonnet';
local genesis = { genesis: {} };
{
  'ibc-0': ibc['ibc-0'] + genesis,
  'ibc-1': ibc['ibc-1'] + genesis,
}
```

Reply:
I've changed the config. Now the config has 3 chains to test multi-hop nft transfers.